jax/_src/scipy/stats/_core.py

```diff
@@ -148,11 +170,25 @@ def rankdata(
     return .5 * (count[dense] + count[dense - 1] + 1).astype(dtypes.canonicalize_dtype(jnp.float_))
   raise ValueError(f"unknown method '{method}'")
 
-@implements(scipy.stats.sem, lax_description="""\
-Currently the only supported nan_policies are 'propagate' and 'omit'
-""")
+
 @partial(jit, static_argnames=['axis', 'nan_policy', 'keepdims'])
 def sem(a: ArrayLike, axis: int | None = 0, ddof: int = 1, nan_policy: str = "propagate", *, keepdims: bool = False) -> Array:
+  """Compute the standard error of the mean.
```

Comment:
Not being a statistician, I had to go read https://en.wikipedia.org/wiki/Standard_error to figure this one out also.

I didn't take enough college stats courses!